### ISSUE TITLE
Filenames are very lossy

### ISSUE THREAD

**Issue by TheTyrius** [kivaar/long-live#7]
Hi,
I wonder why you opted to name files only by rising numbers? In the printed output I see something like "original_filename.jpg is saved as XXX.jpg". 

For my gallery images[i] seems to work perfectly as a filename to preserve the information. Do some galleries not have a nice filename?

Cheers and thanks for the tool!

**Comment by kivaar** [comment]
Hey there, good question!

You're right - I did this to preserve file order because the gallery I was downloading from when I made this script had numbering which was not ideal. For example, once downloaded, it would look like:

1.jpg
10.jpg
11.jpg
12.jpg
...
2.jpg
21.jpg

This isn't the case with every gallery so it would be a good idea to have a prompt (or checkbox in the GUI) to choose between original filenames or the current numbering system.

Feel free to have a crack at it if you like!
